coalib/bearlib/aspects/Formatting.py

```diff
@@ -0,0 +1,393 @@
+from coalib.bearlib.aspects import Root, Taste
+
+
+@Root.subaspect
+class Formatting:
+    """
+    Formatting refers to the visual appearance of you source code.
+    """
+    class docs:
+        example = """
+        # Here is an example of python code with lots of
+        # formatting issues including: trailing spaces, missing spaces
+        # around operators, incorrect indentation etc...
```

Comment:
'incorrect' is vague, and could mean invalid or inconsistent or just strange.
This is a critical wording issue as 'invalid' whitespace is a syntax error aspect, not a formatting aspect. We must be clear about this distinction.